src/components/InstancesList/InstancesList.js

```diff
@@ -410,6 +422,12 @@ class InstancesList extends React.Component {
             onClickHandler: buildOnClickHandler(this.triggerQuickExport),
             isDisabled: !selectedRowsCount || isQuickExportLimitExceeded,
           })}
+          {this.getActionItem({
+            id: 'dropdown-clickable-import-record',
+            icon: 'lightning',
```

Comment:
You make an excellent point. No, there was never a UX mockup for this. I just picked the icon from our selection that seemed the least inappropriate. If you have a better suggestion, I am ready to hear it.

(For what it's worth, I don't think "Fast Add" existed when I started this work, so two separate but concurrent work streams on the Inventory app both chose the same icon for their menu items.)

Reply:
Good question. If you wanted, you could ask in the #ux channel. Alternately...take the existing download icon, rotate it 180, and submit it as a PR for an `upload` icon and use that. 😆

Reply:
If you think I am getting involved in any more PRs than I strictly need to, you're crazy! :-)

In any case, this is not an upload: that would be pushing something up from the user's computer to the Internet (where a download is pulling something down from the Internet to the user's computer). This is a sort of sideload, where something on the Internet pulls something from elsewhere on the Internet. It's not at all clear what a good icon for this would look like.

Reply:
Rotate it 90 degrees? 🤣